File: ios/Plugin/IntentionallyCrashPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(IntentionallyCrashPlugin, "IntentionallyCrash",
           CAP_PLUGIN_METHOD(crash, CAPPluginReturnPromise);
)

File: ios/Plugin/IntentionallyCrashPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(IntentionallyCrashPlugin)
public class IntentionallyCrashPlugin: CAPPlugin {
    @objc public func crash() {
        fatalError()
    }
}
